psi4/driver/driver_nbody.py

```diff
@@ -358,6 +358,10 @@ def nbody_gufunc(func: Union[str, Callable], method_string: str, **kwargs):
     elif metadata['ptype'] == 'hessian':
         core.set_variable("CURRENT HESSIAN", nbody_results['ret_ptype'])
 
+    nbody_number = sum(len(v) for k, v in metadata["compute_dict"]["all"].items())
+    core.set_variable("NBODY NUMBER", nbody_number)
```

Comment:
I can't find any reference to this variable in existing Psi. Is this a new variable?

Reply:
Yes, I've put "cbs|nbody|findif number" vars in ddd as a way to check the wrappers are planning correctly. so pre-porting those to master. need to tell the docs.